__docs__/wonder-blocks-pill/pill.stories.tsx

```diff
@@ -245,7 +245,8 @@ export const WithStyle: StoryComponentType = () => {
 
         ":active": {
             outlineColor: tokens.semanticColor.core.border.neutral.subtle,
-            backgroundColor: tokens.semanticColor.surface.overlay,
```

Comment:
This wasn't being used as an overlay so we can used a strong neutral background token instead!